### PR TITLE
chore: updating service packages according to ts-server boilerplate

### PR DESCRIPTION
**Pull request by almog8k**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                         |

Further  information:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,71 +39,71 @@
...
     "@map-colonies/error-types": "^1.1.5",
-    "@map-colonies/express-access-log-middleware": "^1.0.0",
-    "@map-colonies/js-logger": "^0.0.5",
+    "@map-colonies/express-access-log-middleware": "^2.0.1",
+    "@map-colonies/js-logger": "^1.0.1",
     "@map-colonies/mc-model-types": "^14.0.1",
-    "@map-colonies/mc-priority-queue": "^5.0.1",
-    "@map-colonies/mc-utils": "^1.8.1",
-    "@map-colonies/openapi-express-viewer": "^2.0.1",
+    "@map-colonies/mc-priority-queue": "^6.0.0",
```
Comment (asafMasa) [inline package.json]: 6.1.0

File: src/containerConfig.ts [hunk 2]
```diff
@@ -29,22 +28,16 @@ export const registerExternalValues = (options?: RegisterOptions): DependencyCon
...
     tilesTaskType: externalClientsConfig.exportJobAndTaskTypes.taskFinalizeType,
   };
-  // @ts-expect-error the signature is wrong
   // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
   const logger = jsLogger({ ...loggerConfig, prettyPrint: loggerConfig.prettyPrint, hooks: { logMethod } });
 
-  const metrics = new Metrics(SERVICE_NAME);
-  const meter = metrics.start();
-
-  tracing.start();
```
Comment (asafMasa) [inline src/containerConfig.ts:39]: why was it removed?

**Review by asafMasa — APPROVED** [review]

File: src/containerConfig.ts (hunk 2 above)
Comment (almog8k) [reply src/containerConfig.ts:39]: restored

File: package.json (hunk 1 above)
Comment (almog8k) [reply package.json]: upgraded

**Review by asafMasa — APPROVED** [review]

**Pull request merged**